shell/pom.xml

```diff
@@ -25,6 +25,11 @@
       <artifactId>guava</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.reflections</groupId>
+      <artifactId>reflections</artifactId>
+      <version>0.9.10</version>
+    </dependency>
```

Comment:
for now there're 30+ commands, which is a big number to enumerate. I think the idea of this change is to reduce that kind of boilerplates.